mia/medium/super-famicom.cpp

```diff
@@ -561,7 +560,8 @@ auto SuperFamicom::board() const -> string {
   //Bishoujo Senshi Sailor Moon SuperS - Fuwafuwa Panic (Japan)
   //so we identify it with this embedded string
   string sufamiSignature = "BANDAI SFC-ADX";
-  if(string(data.view(0, sufamiSignature.length())) == sufamiSignature) board.append("ST-", mode);
+  auto romSignature = string_view(rom.data(), sufamiSignature.length());
```

Comment:
```suggestion
  auto romSignature = string_view((const char*)rom.data(), sufamiSignature.length());
```
`rom.data()` is type `const unsigned char*`, so we need an explicit cast or else we will hit the wrong `string_view` constructor.